Firmware/MotorControl/controller.cpp

```diff
@@ -128,19 +128,19 @@ bool Controller::update(float* torque_setpoint_output) {
             ? vel_estimate_src_ : nullptr;
 
     // Calib_anticogging is only true when calibration is occurring, so we can't block anticogging_pos
-    float anticogging_pos = axis_->encoder_.pos_estimate_ / axis_->encoder_.getCoggingRatio();
+    float anticogging_pos = axis_->encoder_.pos_est_rad_ / axis_->encoder_.getCoggingRatio();
     if (config_.anticogging.calib_anticogging) {
         if (!axis_->encoder_.pos_estimate_valid_ || !axis_->encoder_.vel_estimate_valid_) {
             set_error(ERROR_INVALID_ESTIMATE);
             return false;
         }
         // non-blocking
-        anticogging_calibration(axis_->encoder_.pos_estimate_, axis_->encoder_.vel_estimate_);
+        anticogging_calibration(axis_->encoder_.pos_est_rad_, axis_->encoder_.vel_est_rad_);
     }
 
     // TODO also enable circular deltas for 2nd order filter, etc.
     if (pos_wrap_src_) {
-        float cpr = *pos_wrap_src_;
+        float cpr = *pos_wrap_src_ * 2.0f * M_PI / ((float)axis_->encoder_.config_.cpr);
```

Comment:
I'm not sure the float cast is needed here, but in general prefer static_cast<float>() to (float)()

Reply:
I removed the C-style casts I added if it would have been implicitly done anyway.

Reply:
I disagree. I think the (float) are good for clarity that we are converting int to float here. But `static_cast` is too verbose and hard to read. All this code is on POD anyway, so there is no problem using (float).

Reply:
I put the `(float)'s` back.